<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDatabaseEditDialog</class>
 <widget class="QDialog" name="KeyDatabaseEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Key Database Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="keyDBNameLabel">
       <property name="text">
        <string>Key Database Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="keyDBNameLineEdit">
       <property name="maxLength">
        <number>18</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="keyDBPathLabel">
       <property name="text">
        <string>Key Database Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="keyDBPathShowLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectKeyDBButton">
       <property name="text">
        <string>Select A Key Database Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="convert2RelativePathCheckBox">
       <property name="text">
        <string>Convert to Relative Path</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyDatabaseEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
